### PR TITLE
FInd Function

### PR DESCRIPTION
**Pull request by deaspo**
@dannyparsons am still polishing up the code, not yet ready for merging

### CODE REVIEW

**Comment by deaspo** [comment]
@dannyparsons  ready for review

File: instat/frmLog.vb [hunk 1]
```diff
@@ -18,4 +18,12 @@
     Private Sub frmLog_VisibleChanged(sender As Object, e As EventArgs) Handles Me.VisibleChanged
         frmMain.mnuViewLog.Checked = Me.Visible
     End Sub
+
+    Private Sub txtLog_GotFocus(sender As Object, e As EventArgs) Handles txtLog.GotFocus
+        frmMain.mnuEditFind.Enabled = True
+    End Sub
+
+    Private Sub txtLog_LostFocus(sender As Object, e As EventArgs) Handles txtLog.LostFocus
+        frmMain.mnuEditFind.Enabled = False
```
Comment (dannyparsons) [inline instat/frmLog.vb]: I don't think we need this. Find should always be enabled.

**Comment by dannyparsons** [comment]
What are the two .orig files?

**Comment by deaspo** [comment]
@dannyparsons  made the changes

**Pull request merged**